tests/test_routes.py

```diff
@@ -34,7 +34,10 @@ def tearDownClass(cls):
         db.session.close()
 
     def setUp(self):
-        """ This runs before each test """
+        """Runs before each test"""
+        db.session.query(Shopcart).delete()  # clean up the last tests
```

Comment:
line 38 and line 39 duplicate with line 42 and line 43, can you help remove them?

Reply:
sure, sorry about the duplicate